docs/developing.md

```diff
@@ -32,6 +32,16 @@ Lint Workload and other YAML:
 ./run-genny lint-yaml
 ```
 
+### Workload Documentation Generation
+
+Markdown documentation is generated for the workload YAML files. When a workload is updated run the following command:
```

Comment:
~do we automatically regenerate a new docs suite whenever a workload changes?~

Based on ```f_check_generated_docs``` it looks like we don't but the merge fails until you run it.

Reply:
I prefer this UX to me forgetting and having broken docs! Automated would be better, but there's no way to guarantee that it is run.